cvrf2csaf/section_handlers/document_notes.py

```diff
@@ -3,11 +3,52 @@
 
 
 class DocumentNotes(SectionHandler):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, config):
+        super().__init__(config=config)
+
+        self.enum_categories = {
+            'description',
+            'details',
+            'faq',
+            'general',
+            'legal_disclaimer',
+            'other',
+            'summary'
+        }
+
+    def _process_mandatory_and_optional(self, root_element):
+        notes = []
+
+        for elem_note in root_element.Note:
+
+            # mandatory
+            new_note = dict(
+                text=elem_note.text,
+                category=elem_note.get('Type'),
+            )
+
+            if new_note['category'].lower() not in self.enum_categories:
+                log_msg = f'Invalid document notes category ' \
+                          f'{new_note["category"]}. ' \
+                          f'Should be one of: {",".join(str(x) for x in sorted(self.enum_categories))}!'
+                if self.config.get('force', False):
```

Comment:
Here is a place, where we would profit from the unconditional set value, to not have to use get here and elsewhere, and also, to not surprise a user who set the value `foo`  or any other non-empty string in her config file

Reply:
Yes. we will recheck the transformation from `foo` into `false` during the initial config read.